crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1441,7 +1481,7 @@ where
         let tx_digest = *certificate.digest();
         let timeout_after_quorum = self.timeouts.post_quorum_timeout;
 
-        let cert_ref = &certificate;
+        let cert_ref = certificate;
```

Comment:
Is the `cert_ref` needed anymore? I believe you can just used `certificate` instead